tests/unittest_util.c

```diff
@@ -66,9 +66,6 @@ getTempFilename (void)
   }
 
   g_close (fd, NULL);
-  if (g_remove (tmp_fn) != 0) {
-    _print_log ("failed to remove temp file %s", tmp_fn);
-  }
 
```

Comment:
The users expect that `getTempFilename()` function returns a string without creating the file.
So if you want to remove this line, what about adding some comments as below?
This function is used for test so it looks OK.

```cpp
* @return file name (should finalize it with g_remove() and g_free() after use)
```

Reply:
Thanks for the suggestion. Updated.